src/backend/distributed/sql/udfs/citus_isolation_test_session_is_blocked/latest.sql

```diff
@@ -26,30 +26,7 @@ RETURNS boolean AS $$
     RETURN EXISTS (
       SELECT 1 FROM citus_internal_global_blocked_processes()
         WHERE waiting_global_pid = mBlockedGlobalPid
-    ) OR EXISTS (
-      -- Check on the workers if any logical replication job spawned by the
-      -- current PID is blocked, by checking it's application name
-      -- Query is heavily based on: https://wiki.postgresql.org/wiki/Lock_Monitoring
```

Comment:
why did we have this?

Reply:
Because it previously was the only way to detect if creating a logical replication slot was blocked. These slots would be created automatically by creating the subscription and the application name would include the subscription name. In the subscription name we included the pid, so we could use it in this function to see if it was blocked.

Now this isn't necessary anymore, because we create the replication slots manually ourselves and thus it can use regular globalPID based block detection.

Reply:
I also removed the part of the code where we add the pid to the subscription name in isolation tests now.